Frameworks/CoreGraphics/CGImage.mm

```diff
@@ -376,6 +369,55 @@ CGImageRef CGImageCreateWithMaskingColors(CGImageRef image, const CGFloat* compo
 
 #pragma region WIC_HELPERS
 
+IWICBitmap* _CGImageGetImageSource(CGImageRef image) {
+    RETURN_NULL_IF(!image);
+    return image->ImageSource().Get();
+}
+
+DisplayTexture* _CGImageGetDisplayTexture(CGImageRef image) {
+    RETURN_NULL_IF(!image);
+    CGIWICBitmap* iwicImg = dynamic_cast<CGIWICBitmap*>(image->ImageSource().Get());
+    RETURN_NULL_IF(!iwicImg);
```

Comment:
We need to queryinterface with ID. implementing it.

---
In reply to: [87101932](https://github.com/Microsoft/WinObjC/pull/1332#discussion_r87101932) [](ancestors = 87101932)